gcloud/bigtable/row_data.py

```diff
@@ -147,6 +147,19 @@ def clear(self):
         self._chunks_encountered = False
         self._cells.clear()
 
+    def update_from_read_rows(self, read_rows_response_pb):
+        """Updates the current row from a ``ReadRows`` response.
+
+        :type read_rows_response_pb:
+            :class:`._generated.bigtable_service_messages_pb2.ReadRowsResponse`
+        :param read_rows_response_pb: A response streamed back as part of a
+                                      ``ReadRows`` request.
+
+        :raises: :class:`NotImplementedError <exceptions.NotImplementedError>`
+                 always
+        """
+        raise NotImplementedError
+
```

Comment:
Do we need a message explaining why we can't implement this method, or indicating what the caller should do instead?